From 32725f06d25885b69396e4729d199476a54d1739 Mon Sep 17 00:00:00 2001
From: Chris <chris.dixon.78.burnbridge@gmail.com>
Date: Fri, 08 Aug 2026 18:35:00 +0100
Subject: [PATCH] let the environment choose GL_LIB

A system whose glvnd provides libOpenGL rather than libGL can then name it
without losing the -lEGL that GLEW_EGL adds.
---
--- a/libretro/Makefile
+++ b/libretro/Makefile
@@ -70,7 +70,7 @@
 		GLES = 1
 		GL_LIB := -lGLESv2
 	else
-		GL_LIB := -lGL
+		GL_LIB ?= -lGL
 		ifeq ($(GLEW_EGL),1)
 			GL_LIB += -lEGL
 		endif
